extensions/notebook/package.json

```diff
@@ -666,7 +666,8 @@
     "tar": "^6.0.1",
     "vscode-languageclient": "^5.3.0-next.1",
     "vscode-nls": "^4.0.0",
-    "ws": "^7.2.0"
+    "ws": "^7.2.0",
+    "js-yaml": "^3.12.1"
```

Comment:
should this be a dev dependency instead (if this is just for tests)?

Reply:
We actually need js-yaml for reading and modifying the toc from books, but it's using the package from the core node_modules directory ( /azuredatastudio/node_modules/js-yaml/lib/js-yaml), so I'm adding it to prevent any future issues.

Reply:
It's nice to keep these in alphabetical order